cluster/delegate.go

```diff
@@ -25,8 +25,12 @@ import (
 )
 
 // Maximum number of messages to be held in the queue.
```

Comment:
This comment would need to be moved below `const (`.

Reply:
done